src/app/dashboard/hero-card/hero-card.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="hero" *ngIf="hero" [class.active]="isActive" (click)="toggleHero.emit(!isActive)">
+  <img [src]="hero.imageSrc" alt="avatar hero" width="100" height="100">
+  <div class="hero-info">
+    <p class="hero-info-name"><b [title]="hero.name">{{ hero.name }}</b></p>
+    <p><b>Health</b>: {{ hero.health }}</p>
```

Comment:
Unfortunately the health is not going down as they battle

Reply:
My idea is hero's health in the battle do not impact to hero's health in list.